Sources/GoTrue/GoTrueLocalStorage.swift

```diff
@@ -30,3 +32,37 @@ struct KeychainLocalStorage: GoTrueLocalStorage {
     try keychain.remove(key)
   }
 }
+#else
+// Secure storage in Linux seems to be quite different than say macOS and Windows
+// The keyctl API set exists, but it doesn't persist across reboots like the
+// keychain on macOS and the password vault (wincred.h) on Windows does. So this
+// provides a bit of a challenge of how to implement this on Linux.
+//
+// Realistically, you probably want to pull in gpg and build your own implementation.
+final class KeychainLocalStorage: GoTrueLocalStorage {
```

Comment:
I'd prefer to force the developer to pass a `GoTrueLocalStorage` implementation on client initialization in case of a non-darwin platform instead of providing a default implementation.